### ISSUE TITLE
[a11y] Cannot open Preparations subview

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1279]
**Describe the bug**
I get this error whenever attempting to open the Preparations subview:
**Error: Value is not defined**

```
SubView@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:268558
Ve@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:270107
div
DataEntry.Cell
div
div
DataEntry.Grid@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:41564
div
C@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:226720
k@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:226428
div
DataEntry.Cell
div
div
div
DataEntry.Grid@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:41564
fieldset
DataEntry.SubForm
f@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:85816
h@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:89711
D@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:193566
u@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:271677
SubView@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:268558
Ve@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:270107
div
DataEntry.Cell
div
div
DataEntry.Grid@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:41564
div
C@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:226720
k@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:226428
form
Form
section
Container.Generic
q@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:212390
$@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:213743
H@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:216605
s@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:187900
g@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:77704
b@https://a11y-2-2-2.test.specifysystems.org/static/js/main.5c8077cb6b42ca69f165.bundle.js:2:64467
```

**To Reproduce**
1. Go to `newmexico` database
2. Try to add a preparation from the CO form

It works correctly on `edge`

macOS, Safari

**Comment by maxxxxxdlp** [comment]
Should be fixed now as I can not reproduce the issue anymore
